flood_ml/requirements.txt

```diff
@@ -0,0 +1,41 @@
+absl-py==2.1.0
+astunparse==1.6.3
+black==24.3.0
+certifi==2024.2.2
+charset-normalizer==3.3.2
+click==8.1.7
+flatbuffers==24.3.25
+gast==0.5.4
+google-pasta==0.2.0
+grpcio==1.62.1
+h5py==3.10.0
+idna==3.6
+keras==3.1.1
```

Comment:
Even if Tensorflow versions come with built-in Keras, should we still specify keras as requirement?

Reply:
I didn't install Keras separately. The TF install includes Keras and many other packages, and `pip freeze` includes all of them.